functions.sh

```diff
@@ -1993,7 +2033,9 @@ make_swraid() {
 
         # workaround: the 2>&1 redirect is valid syntax in shellcheck 0.4.3-3, but not in the older version which is currently used by travis
         # shellcheck disable=SC2069
-        yes | mdadm -q -C "$raid_device" -l"$array_raidlevel" -n"$n" "$array_metadata" "$can_assume_clean" "$components" 2>&1 >/dev/null | debugoutput ; EXITCODE=$?
+        # yes | mdadm -q -C "$raid_device" -l"$array_raidlevel" -n"$n" "$array_metadata" "$can_assume_clean" "${components[@]}" 2>&1 >/dev/null | debugoutput ; EXITCODE=$?
```

Comment:
do we want to keep the old version or just throw it away? We still have it in the git history.